homeassistant/components/matter/device_platform.py

```diff
@@ -0,0 +1,24 @@
+"""All mappings of Matter devices to Home Assistant platforms."""
```

Comment:
Maybe it's just me but wouldn't "device_mappings.py" be a more descriptive filename instead of device_platform ?